.gitlab-ci.yml

```diff
@@ -164,6 +164,17 @@ Tools Tests:
     - make clean repo-setup ENV=ci
     - make tools-test
 
+Benchmark Build:
+  stage: test
```

Comment:
suggestion/ As we don't test any logic in "Benchmark Build" it rather belongs to `smoke-test` stage.

Other than changing the ordering of this job and delaying pipeline termination on failure, it will have no further impact tho.